internal/pkg/agent/application/info/svc_windows.go

```diff
@@ -21,9 +23,10 @@ func RunningUnderSupervisor() bool {
 	if err != nil {
 		return false
 	}
-	defer windows.FreeSid(serviceSid)
+	defer func() { _ = windows.FreeSid(serviceSid) }()
```

Comment:
unrelated change, fixing lint error: Error return value of `windows.FreeSid` is not checked (errcheck)